cluster-autoscaler/cloudprovider/azure/README.md

```diff
@@ -185,11 +185,8 @@ A configurable jitter (`AZURE_VMSS_VMS_CACHE_JITTER` environment variable, defau
 | vmssVmsCacheTTL | 300 | AZURE_VMSS_VMS_CACHE_TTL | vmssVmsCacheTTL |
 | vmssVmsCacheJitter | 0 | AZURE_VMSS_VMS_CACHE_JITTER | vmssVmsCacheJitter |
 
-The `AZURE_ENABLE_DYNAMIC_INSTANCE_LIST` environment variable enables workflow that fetched SKU information dynamically using SKU API calls. By default, it uses static list of SKUs.
```

Comment:
@comtalyst is it preferred to "deprecate" this by removing it from code references while keeping it w/ a "DEPRECATED" message in the README and azure_config.go?

It feels like this would be an ok flag/env var to just completely remove rather than adding these DEPRECATED messages.

Reply:
Given it's just beed added, I don't mind just completely removing it